components/places/android/src/main/java/mozilla/appservices/places/PlacesConnection.kt

```diff
@@ -1095,31 +1100,23 @@ fun stringOrNull(jsonObject: JSONObject, key: String): String? {
 }
 
 data class SearchResult(
-    val searchString: String,
     val url: String,
     val title: String,
-    val frecency: Long,
-    val iconUrl: String? = null
+    val frecency: Long
     // Skipping `reasons` for now...
```

Comment:
@thomcc I see we are still skipping `reasons` despite "proto-encoding" it, do we want to expose that?

Reply:
It doesn't matter much but yes we probably should include it.